e2e/cypress/integration/interactive_dialog/full_dialog_spec.js

```diff
@@ -31,6 +31,8 @@ const optionsLength = {
 
 describe('Interactive Dialog', () => {
     before(() => {
+        cy.requireWebhookServer();
+
```

Comment:
Getting failure in this test
![Screen Shot 2020-03-10 at 6 53 14 PM](https://user-images.githubusercontent.com/487991/76374464-81f9c100-6300-11ea-9bef-c77144587e6e.png)

Reply:
I believed it's not related.  I suggests to address it separately from this PR.  Same on another comment.